.circleci/config.yml

```diff
@@ -45,8 +32,6 @@ commands:
           env_var: APP_VERSION
```

Comment:
The above step `set_app_version` has been deleted, so where does this value come from now?

Reply:
Defo still there, line 68